### ISSUE TITLE
Ensure white text is visible in CKEditor

### ISSUE THREAD

**Issue by akslay** [az-digital/az_quickstart#2896]
## Problem/Motivation
When using white text in the CKEditor interface, you are unable to see the content unless you highlight it. This should be fixed for usability and accessibility.

### Describe the bug
White text used in the editor is not visible (while editing content).

### To Reproduce
Use the `.text-white` or `.btn-outline-white` class on your content within the editor.

## Proposed resolution
- Add a CSS rule to `/themes/custom/az_barrio/css/ckeditor5.css` which adds a background-color to white text in CKEditor to keep it visible.
- Add help text indicating white text will be "highlighted" in gray so it remains visible in the editor.
- Document and announce this change to editors so the color change in the editor isn't unexpected.

### Expected behavior
Users should be able to read their content, while editing, without highlighting their content in the CKEditor.

##  Screenshots
Current:
<img width="250" alt="Screenshot 2023-11-01 at 4 08 04 PM" src="https://github.com/az-digital/az_quickstart/assets/52931751/adf4cb0f-5132-44c4-922c-08b37706d122">
<img width="250" alt="Screenshot 2023-11-01 at 4 09 19 PM" src="https://github.com/az-digital/az_quickstart/assets/52931751/7fee6fae-13ab-41d2-9a58-69d95f53de39">

Suggested improvement:
<img width="250" alt="Screenshot 2023-11-01 at 4 08 59 PM" src="https://github.com/az-digital/az_quickstart/assets/52931751/602a6052-d246-4f63-abdb-e9cb052aea7a">
